fastestimator/estimator/trace.py

```diff
@@ -1045,27 +1076,25 @@ class ModelCheckpoint(Trace):
 
     Args:
         save_dir (str): The directory to save the trained models.
-        model_names (list): The list of models to save. If not proveded, save all available models.
+        model (list): The list of models to save. If not proveded, save all available models.
         monitor_name (str): The trace name to be monitored when `save_best_only=True`.
         mode (str): Save models during either `train` or `eval`. Default is `eval`.
-        verbose (int): verbosity mode, 0 or 1
         save_best_only (bool): Keep only the best model.
         save_mode: Can be `'min'`, `'max'`, or `'auto'`.
         save_freq: Number of epochs to save models. Cannot be used with `save_best_only=True`.
     """
     def __init__(self,
                  save_dir,
-                 model_names=None,
+                 models,
```

Comment:
is models really more intuitive than model_names here? if so, the doc string needs to say 'models' rather than 'model'

Reply:
fixed.